packages/plugin-essentials/sources/commands/unlink.ts

```diff
@@ -49,41 +47,43 @@ export default class UnlinkCommand extends BaseCommand {
     const topLevelWorkspace = project.topLevelWorkspace;
     const workspacesToUnlink = new Set<string>();
 
-    if (!this.leadingArgument && this.all) {
+    if (!this.leadingArguments.length && this.all) {
```

Comment:
We tend to use explicit length checks in the codebase:

```suggestion
    if (this.leadingArguments.length === 0 && this.all) {
```

Reply:
Is it worth adding this as a lint rule?

Reply:
It is, I think typescript-eslint has a rule requiring strict boolean expressions or something similar that would cover this case.